HymnsTests/Display/DisplayHymnBottomBarViewModelSpec.swift

```diff
@@ -1,22 +1,85 @@
 import Quick
 import Nimble
+import Mockingbird
+import Combine
 @testable import Hymns
 
 // swiftlint:disable:next type_name
 class DisplayHymnBottomBarViewModelSpec: QuickSpec {
 
     override func spec() {
         describe("DisplayHymnBottomBarViewModel") {
-
+            let testQueue = DispatchQueue(label: "test_queue")
+            var hymnsRepository: HymnsRepositoryMock!
             var target: DisplayHymnBottomBarViewModel!
 
+            beforeEach {
+                hymnsRepository = mock(HymnsRepository.self)
+            }
+
             describe("init") {
                 beforeEach {
-                    target = DisplayHymnBottomBarViewModel(hymnToDisplay: classic1151)
+                    target = DisplayHymnBottomBarViewModel(hymnToDisplay: classic1151, hymnsRepository: hymnsRepository, mainQueue: testQueue, backgroundQueue: testQueue)
                 }
                 it("should set up the info dialog") {
                     expect(target.songInfo).to(equal(SongInfoDialogViewModel(hymnToDisplay: classic1151)))
                 }
+                it("shareable lyrics should be an empty string") {
+                    expect(target.shareableLyrics).to(equal(""))
+                }
+            }
+
+            context("with valid repository and hymn 1151") {
+                beforeEach {
+                    target = DisplayHymnBottomBarViewModel(hymnToDisplay: classic1151, hymnsRepository: hymnsRepository, backgroundQueue: testQueue)
+                    given(hymnsRepository.getHymn(classic1151)) ~> { _ in
+                        Just(nil).assertNoFailure().eraseToAnyPublisher()
+                    }
+                }
+                describe("performing fetch") {
+                    beforeEach {
+                        target.fetchLyrics()
+                        testQueue.sync {}
+                        testQueue.sync {}
+                        testQueue.sync {}
+                    }
+                    it("should call hymnsRepository.getHymn") {
+                        verify(hymnsRepository.getHymn(classic1151)).wasCalled(exactly(1))
+                    }
+                    it("should set up the info dialog") {
+                        expect(target.songInfo).to(equal(SongInfoDialogViewModel(hymnToDisplay: classic1151)))
+                    }
```

Comment:
You don't really need this because the fetch doesn't affect the song info dialog